firebase-installations-interop/api.txt

```diff
@@ -1,6 +1,17 @@
 // Signature format: 2.0
 package com.google.firebase.installations {
 
+  public interface FirebaseInstallationsApi {
```

Comment:
I think you'd want to hide this interface and instead have `FirebaseInstallations` exposed. This is consistent with how iid has it documented.